form/form_customization.rst

```diff
@@ -735,6 +735,35 @@ class to the ``div`` element around each row:
         </div>
     {% endblock form_row %}
 
+.. tip::
+
+    If you want to customize some instances of the same form only (without
+    the need to create a new form type) you can set the ``block_prefix``
+    option in your form type::
+
+        use Symfony\Component\Form\Extension\Core\Type\TextType;
+        use Symfony\Component\Form\FormBuilderInterface;
+
+        public function buildForm(FormBuilderInterface $builder, array $options)
+        {
+            $builder->add('name', TextType::class, array(
+                'block_prefix' => 'wrapped_text',
+            ));
+        }
+
+    .. versionadded:: 4.3
```

Comment:
please add a blank line after the directive

Reply:
I'm not sure, it doesn't seem a common practice within the documentation for `versionadded` directive.

Reply:
It will be 😄 trust me ;-)

Reply:
Refs https://github.com/symfony/symfony-docs/pull/10823

Reply:
Done, thanks!